nemo/collections/nlp/data/language_modeling/megatron/gpt_sft_chat_dataset.py

```diff
@@ -29,18 +29,31 @@
 SYSTEM_TOKEN = "<extra_id_0>System\n"
 TURN_TOKEN = "<extra_id_1>"
 
-GUARD_RAIL_INSTRUCTION = {
+TYPE_INSTRUCTION = {
     "TEXT_TO_CANONICAL_FORM": "Given a dialogue, for each turn you need to generate a short summary called a canonical form. Generate the canonical form for the last turn in the dialogue.",
```

Comment:
Are we still going to make use of canonical_forms? Otherwise, may remove it? I think in general perhaps having fewer options is better, otherwise we might risk confusing others

Reply:
yes we can remove canonical forms.